src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/GATKSVVCFConstants.java

```diff
@@ -64,30 +66,49 @@
     public static final String INV33 = "INV33";
     public static final String INV55 = "INV55";
 
+    // type specific: CPX
+    public static final String CPX_SV_SYB_ALT_ALLELE_STR = "CPX";
+    public static final String CPX_EVENT_ALT_ARRANGEMENTS = "ALT_ARRANGEMENT";
+    public static final String CPX_SV_REF_SEGMENTS = "SEGMENTS";
+    public static final String CPX_EVENT_KEY = "CPX_EVENT";
+
     // not defined in output vcf header but used in internal id that is currently output in the ID column
     public static final String INTERVAL_VARIANT_ID_FIELD_SEPARATOR = "_";
-
     public static final String DUP_TAN_CONTRACTION_INTERNAL_ID_START_STRING = "DEL-DUPLICATION-TANDEM-CONTRACTION";
     public static final String DUP_TAN_EXPANSION_INTERNAL_ID_START_STRING = "INS-DUPLICATION-TANDEM-EXPANSION";
     public static final String DUP_INV_INTERNAL_ID_START_STRING = "INS-DUPLICATION-INVERTED-EXPANSION";
 
-    public static final String EXTERNAL_CNV_CALLS = "EXTERNAL_CNV_CALLS";
 
+    // format block
     public static final String COPY_NUMBER_FORMAT = "CN";
     public static final String COPY_NUMBER_QUALITY_FORMAT = "CNQ";
 
-    public static final String CPX_SV_SYB_ALT_ALLELE_STR = "CPX";
-    public static final String CPX_EVENT_ALT_ARRANGEMENTS = "ALT_ARRANGEMENT";
-    public static final String CPX_SV_REF_SEGMENTS = "SEGMENTS";
-
-    public static final String CTG_GOOD_NONCANONICAL_MAPPING = "CTG_GOOD_NONCANONICAL_MAPPING";
-
-    public static final List<String> expectedHeaderLinesInVCF
-            = Stream.of("SVTYPE", "SVLEN", "MATEID", "INV", "DEL", "INS", "DUP", "DUP:INV",
-                    "CIPOS", "CIEND", "IMPRECISE", "READ_PAIR_SUPPORT", "SPLIT_READ_SUPPORT", "LINK",
-                    "CTG_NAMES", "TOTAL_MAPPINGS", "MAPPING_QUALITIES", "HQ_MAPPINGS", "ALIGN_LENGTHS", "MAX_ALIGN_LENGTH",
-                    "SEQ_ALT_HAPLOTYPE", "INSSEQ", "INSLEN", "INSSEQ_MAP", "HOMSEQ", "HOMLEN", "DUP_REPEAT_UNIT_REF_SPAN",
-                    "DUP_SEQ_CIGARS", "DUP_NUM", "DUP_ANNOTATIONS_IMPRECISE", "CONTRACTION", "EXPANSION", "DUP_ORIENTATIONS",
-                    "INV33", "INV55", "EXTERNAL_CNV_CALLS", "DUP_IMPRECISE_AFFECTED_RANGE", "CTG_GOOD_NONCANONICAL_MAPPING")
-                    .sorted().collect(Collectors.toList());
+    // filter block
+    public static final String ASSEMBLY_BASED_VARIANT_MQ_FILTER_KEY = "LOW_MQ";
+    public static final String ASSEMBLY_BASED_VARIANT_ALN_LENGTH_FILTER_KEY = "SHORT_ALN";
+
+    ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+    public static final Stream<String> expectedAltAlleleHeaderKeysInVCF
```

Comment:
I think that these should be in test code somewhere, not in main code.

Reply:
What if we added some assertions to our integration test that checked whether everything is in the header? We could collect all of the alt alleles and info and format annotation keys we see in the variants in `StructuralVariationDiscoveryPipelineSparkIntegrationTest.svDiscoveryVCFEquivalenceTest` and assert that they have corresponding header line entries. Would that address your concern? You could split that work off of this PR, too, and make a ticket for it.